src/backend/joanie/tests/core/api/order/test_submit_installment_payment.py

```diff
@@ -616,7 +616,7 @@ def test_api_order_submit_installment_payment_order_no_payment_state_with_credit
             },
         )
 
-        self.assertTrue(_mock_create_one_click_payment.called)
+        self.assertTrue(mock_create_one_click_payment.called)
```

Comment:
This is redundant, you are already testing that is method has been called with `assert_called_once_with`

```suggestion
```